### PR TITLE
chore: remove hambach

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#1545

Remove code related to Hambach.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] targeted `dev` branch
- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] submit a PR against `master` branch after this one (and other PRs depending on this one, e.g. on regen-network/regen-web, if applicable) get merged

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by wgwz — APPROVED** [review]
thanks! LGTM

File: server/.env.example [hunk 1]
```diff
@@ -30,12 +30,8 @@ RECAPTCHA_V3_SECRET_KEY=
 # - or to Mainnet on production: https://docs.regen.network/getting-started/live-networks.html#regen-mainnet
 LEDGER_TENDERMINT_RPC=
 LEDGER_REST_ENDPOINT=
-# The following endpoints are supposed to point 
-# to the experimental tesnet, Hambach: https://docs.regen.network/getting-started/live-networks.html#hambach-testnet
-EXP_LEDGER_TENDERMINT_RPC=
-EXP_LEDGER_REST_ENDPOINT=
 # The following endpoints point to the temporary testnet for regen-ledger v4.0 upgrade
-# Once hambach/redwood/mainnet have upgraded to v4.0, we can get rid of this
+# Once redwood/mainnet have upgraded to v4.0, we can get rid of this
```
Comment (blushi) [inline server/.env.example:34]: let's get rid of this 4.0 logic too, both redwood and mainnet being on 5.0

File: server/Server.ts [hunk 2]
```diff
@@ -31,8 +31,6 @@ const REGISTRY_PREVIEW_HOSTNAME_PATTERN =
   /deploy-preview-\d+--regen-registry\.netlify\.app$/;
 const REGISTRY_REDWOOD_HOSTNAME_PATTERN =
   /redwood--regen-registry\.netlify\.app$/;
-const REGISTRY_HAMBACH_HOSTNAME_PATTERN =
-  /hambach--regen-registry\.netlify\.app$/;
 const REGISTRY_V4_HOSTNAME_PATTERN = /v4--regen-registry\.netlify\.app$/;
```
Comment (blushi) [inline server/Server.ts:34]:
```suggestion
```

File: server/Server.ts [hunk 3]
```diff
@@ -50,7 +48,6 @@ const corsOptions = (req, callback): void => {
         originURL.hostname.match(WEBSITE_PREVIEW_HOSTNAME_PATTERN) ||
         originURL.hostname.match(REGISTRY_PREVIEW_HOSTNAME_PATTERN) ||
         originURL.hostname.match(REGISTRY_REDWOOD_HOSTNAME_PATTERN) ||
-        originURL.hostname.match(REGISTRY_HAMBACH_HOSTNAME_PATTERN) ||
         originURL.hostname.match(REGISTRY_V4_HOSTNAME_PATTERN) ||
```
Comment (blushi) [inline server/Server.ts:51]:
```suggestion
```

File: server/Server.ts [hunk 4]
```diff
@@ -118,26 +115,6 @@ app.use(
...
-  app.use(
-    '/exp-ledger-rest',
-    createProxyMiddleware({
-      target: process.env.EXP_LEDGER_REST_ENDPOINT,
-      pathRewrite: { '^/exp-ledger-rest': '/' },
-    }),
-  );
-}
-
 if (process.env.V4_LEDGER_TENDERMINT_RPC) {
```
Comment (blushi) [inline server/Server.ts:118]: ditto, let's remove this

**Review by blushi — APPROVED** [review]

**Pull request merged**